src/ert/simulator/simulation_context.py

```diff
@@ -61,19 +60,9 @@ def _run_forward_model(
             ert.ert_config.preferred_num_cpu,
         )
 
-    queue_evaluators = None
-    if (
-        ert.ert_config.analysis_config.stop_long_running
-        and ert.ert_config.analysis_config.minimum_required_realizations > 0
-    ):
-        queue_evaluators = [
-            partial(
-                job_queue.stop_long_running_jobs,
-                ert.ert_config.analysis_config.minimum_required_realizations,
-            )
-        ]
-
-    asyncio.run(job_queue.execute(evaluators=queue_evaluators))
+    asyncio.run(
+        job_queue.execute(ert.ert_config.analysis_config.minimum_required_realizations)
```

Comment:
Do we want to remove the stop_long_running feature from everest?

Reply:
Nope. Mistake!